cmd/acra-rotate/rotator.go

```diff
@@ -90,13 +94,17 @@ func (rotator *keyRotator) rotateAcrastructWithClientID(clientID, acrastruct []b
 	logger := log.WithFields(log.Fields{"KeyID": string(clientID)})
 	logger.Infof("Rotate AcraStruct")
 	// rotate
-	privateKey, err := rotator.keystore.GetServerDecryptionPrivateKey(clientID)
+	privateKeys, err := rotator.keystore.GetServerDecryptionPrivateKeys(clientID)
 	if err != nil {
 		logger.WithField("acrastruct", hex.EncodeToString(acrastruct)).WithError(err).Errorln("Can't get private key")
 		return nil, err
 	}
-	defer utils.FillSlice(0, privateKey.Value)
-	decrypted, err := base.DecryptAcrastruct(acrastruct, privateKey, nil)
+	defer func() {
+		for _, key := range privateKeys {
+			utils.FillSlice(0, key.Value)
```

Comment:
shouldn't it be `utils.ZeroizePrivateKey(key)`?

Reply:
> shouldn't it be `utils.ZeroizePrivateKey(key)`?

Yeah, I missed the conversion here. Thanks.